src/main/scala/ing/wbaa/druid/DruidResponse.scala

```diff
@@ -22,7 +22,6 @@ import java.time._
 import io.circe._
 import io.circe.java8.time._
 import scala.collection.immutable.ListMap
-import cats.syntax.either._
```

Comment:
Hi @vkorchik
I was about to merge this MR until I saw you removed these imports. They're 'irrelevant' when you compile with Scala 2.12, but with Scala 2.11 the code won't work otherwise. The import provides (amongst others) the `.map` function on on `Either`

If you can undo these removals, we can merge this!

Reply:
@bjgbeelen , done.
But then downgrade project scala version to 2.11.12 to be sure that everything is written can be compiled successfully with the current version of project. Or are there any publishing issues in this case?

Reply:
I think 2.12 should be the default, but in the CI (which is not present anymore, another issue). we should compile for both 2.12 and 2.11 so that we can see there that stuff breaks.